### PR TITLE
Allow user to rename any address

### PR DESCRIPTION
**Pull request by KillariDev**
image

### CODE REVIEW

File: app/ts/components/pages/ConfirmTransaction.tsx [hunk 1]
```diff
@@ -137,13 +138,14 @@ export function ConfirmTransaction() {
...
 	const [currentBlockNumber, setCurrentBlockNumber] = useState<undefined | bigint>(undefined)
-	const [addingNewAddress, setAddingNewAddress] = useState<AddingNewAddressType | 'renameAddressModalClosed'> ('renameAddressModalClosed')
+	const [addingNewAddress, setAddingNewAddress] = useState<InCompleteAddressBookEntry | 'renameAddressModalClosed'> ('renameAddressModalClosed')
 	const [rpcConnectionStatus, setRpcConnectionStatus] = useState<RpcConnectionStatus>(undefined)
 	const [pendingTransactionAddedNotification, setPendingTransactionAddedNotification] = useState<boolean>(false)
 
 	useEffect(() => {
 		async function popupMessageListener(msg: unknown) {
 			const message = ExternalPopupMessage.parse(msg)
+			console.log(message.method)
```
Comment (jubalm) [inline app/ts/components/pages/ConfirmTransaction.tsx:148]: These might be debugging lines

File: app/ts/components/pages/ConfirmTransaction.tsx [hunk 2]
```diff
@@ -166,6 +168,7 @@ export function ConfirmTransaction() {
 				return
 			}
 			if (message.method !== 'popup_update_confirm_transaction_dialog') return
+			console.log('updsd')
```
Comment (jubalm) [inline app/ts/components/pages/ConfirmTransaction.tsx:171]: Remove debug lines

**Review by jubalm — APPROVED** [review]

**Comment by jubalm** [comment]
A screenshot of the addressbook entry after renaming it would help visualize as well.
I would ask @MicahZoltu for opinion on this as to the implication he might see on security standpoint before merging

**Comment by KillariDev** [comment]
> A screenshot of the addressbook entry after renaming it would help visualize as well. I would ask @MicahZoltu for opinion on this as to the implication he might see on security standpoint before merging

there's no addressbook entries for them at the moment visible anymore, so no screenshots available :). That's still work to be done

**Pull request merged**